src/main/java/frc/robot/subsystems/CommandSwerveDrivetrain.java

```diff
@@ -254,25 +262,44 @@ public Command sysIdDynamic(SysIdRoutine.Direction direction) {
         return m_sysIdRoutineToApply.dynamic(direction);
     }
 
-    public SwerveDrivePoseEstimator getPoseEstimator(){
-        return poseEstimator;
+    public SwerveDrivePoseEstimator getFusedPoseEstimator(){
+        return fusedPoseEstimator;
+    }
+
+    public SwerveDrivePoseEstimator getDrivePoseEstimator(){
+        return drivePoseEstimator;
     }
 
     public Rotation2d getRobotRotation(){
         return this.getState().Pose.getRotation();
     } 
     
-    public void updatePoseWithPoseEst(){
-        this.resetPose(poseEstimator.update(
+    public void updateRobotPoseWithFusedPoseEst(){
+        this.resetPose(fusedPoseEstimator.update(
             this.getPigeon2().getRotation2d(), 
             this.getState().ModulePositions)); 
     }
 
+    // doesn't change robot pose, hust stored SwerveDrivePoseEst
+    public void updateDrivePoseWithOdometry(){
+        drivePoseEstimator.update(
+            this.getPigeon2().getRotation2d(),
+            this.getState().ModulePositions);
+    }
+
+    // updates both fused pose and drivetrain only pose
     public void setNewPose(Pose2d newPose){
-        poseEstimator.resetPose(newPose);
+        drivePoseEstimator.resetPose(newPose);
+        fusedPoseEstimator.resetPose(newPose);
         this.resetPose(newPose);
     }
 
+    // resets fusedPoseEst to drivePoseEst
+    public void resetRobotPoseWithDrivePoseEst(){
```

Comment:
Can we use full words in function names, so `Estimation` in this case (and consistently across the pose estimation functions in this PR). I know Java can get a bit verbose in this way but I think it's generally better.